docs/stories/primitives/overview.mdx

```diff
@@ -0,0 +1,5 @@
+import { Title, Meta } from '@storybook/blocks';
+
+<Meta title="Primitives/Overview" parameters={{ viewMode: 'docs' }} />
+
+<Title>Overview</Title>
```

Comment:
Do we need these files?

`docs/stories/primitives/overview.mdx`
`docs/stories/foundations/overview.mdx`

We end up with empty overview pages in storybook which looks a bit strange

Reply:
I'm gonna keep them for now, the storybook in v2 is very much a WIP, we have tickets to amend this sort of stuff, but it's there to start building the shape we want 💪🏼